stock_picking_report_delivery_note_kit/README.rst

```diff
@@ -0,0 +1,25 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+===============================
+Stock picking delivery note kit
+===============================
+
+* Stock picking delivery note kit
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/nuobit/odoo-addons/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Eric Antones <eantones@nuobit.com>
```

Comment:
Add yourself here. In fact, create a readme folder and let the pre-commit do it for you